torq.sh

```diff
@@ -223,6 +221,18 @@ usage() {
   exit 1
  }
 
+if [[ $(printenv | grep SETENV) ]]; then
```

Comment:
I think this can be simplified, something like:

```bash
jmcmurray@homer ~ $ if [[ -z "$SETENV" ]]; then echo "blank"; else echo "not blank"; fi
blank
jmcmurray@homer ~ $ SETENV=test
jmcmurray@homer ~ $ if [[ -z "$SETENV" ]]; then echo "blank"; else echo "not blank"; fi
not blank
```

Reply:
Problem is when I run the following in the script with SETENV in the env var it returns blank. This is why I have done it in the way that I have.